Modules/Scripted/ScreenCapture/ScreenCapture.py

```diff
@@ -220,6 +220,12 @@ def setup(self):
     hbox.addWidget(self.lightboxImageFileNameWidget)
     outputFormLayout.addRow("Output file name:", hbox)
 
+    self.volumeNodeComboBox = slicer.qMRMLNodeComboBox()
```

Comment:
Enable None and Rename options.
Move it to advanced section (in most cases, users would want to write to file and prefer not store in the scene).
Add tooltip that explains what this node is used for.
Only enable this node selector if "single" option is enabled.